astropy/io/tests/test_registry.py

```diff
@@ -506,27 +505,16 @@ class MTable(Table):
         mt['a'].format = '.4f'
         mt['a'].description = 'hello'
 
-        if HAS_YAML:
-            ctx = nullcontext()
-        else:
-            ctx = pytest.warns(
-                AstropyUserWarning,
```

Comment:
lint: ditto `AstropyUserWarning` is no longer used.

Reply:
Done!